delta/rdf/src/main/scala/ch/epfl/bluebrain/nexus/delta/rdf/utils/OrderingJsonKeys.scala

```diff
@@ -0,0 +1,37 @@
+package ch.epfl.bluebrain.nexus.delta.rdf.utils
+
+/**
+  * Ordering for Json keys.
+  */
+trait OrderingJsonKeys extends Ordering[String]
```

Comment:
If it's not too much trouble, I would recommend renaming this to JsonKeyOrdering.